impl/src/main/java/com/sun/faces/cdi/clientwindow/ClientWindowScopeManager.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 1997, 2020 Oracle and/or its affiliates. All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0, which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception, which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ */
+
+package com.sun.faces.cdi.clientwindow;
+
+import jakarta.faces.context.FacesContext;
+import jakarta.servlet.http.HttpSessionEvent;
+import jakarta.servlet.http.HttpSessionListener;
+
+import java.util.logging.Logger;
+
+import static java.util.logging.Level.*;
+
+/**
+ * The manager that deals with non-CDI and CDI ClientWindowScoped beans.
+ */
+public class ClientWindowScopeManager implements HttpSessionListener {
+
+    /**
+     * Stores the logger.
+     */
+    private static final Logger LOGGER = Logger.getLogger(ClientWindowScopeManager.class.getName());
+    /**
+     * Stores the constant to keep track of the ClientWindowScopeManager.
+     */
+    public static final String CLIENT_WINDOW_SCOPE_MANAGER = "com.sun.faces.cdi.clientwindow.clientWindowScopeManager";
+    /**
+     * Stores the CDI context manager.
+     */
+    private ClientWindowScopeContextManager contextManager;
+
+    /**
+     * Constructor.
+     */
+    public ClientWindowScopeManager() {
+        FacesContext context = FacesContext.getCurrentInstance();
+        try {
+            contextManager = new ClientWindowScopeContextManager();
+        } catch (Throwable throwable) {
```

Comment:
please, no

Reply:
Same like existing ViewScopeManager-constructor.
I´ll remove this for both.

Reply:
Did you expect me to remove unused `FacesContext context = FacesContext.getCurrentInstance();` and keep the try-catch block or also remove the try-catch-block?

Reply:
_Just to clarify - I'm not a maintainer of this project - only providing comments. No need to follow them or my expectations._

For the above `try-catch` - I'm only against catching `Throwable`s and continuing the flow like nothing happened, since it would consume `Errors` too.